fido/fido.py

```diff
@@ -344,4 +364,5 @@ def fetch(
         timeout,
         connect_timeout,
         tcp_nodelay,
+        gzip_enabled,
```

Comment:
I believe that if we say gzip_enabled=True somewhere, we should add this request header as well. As it is right now, we never add the header and so I don't see how this code would work unless you pass gzip_enabled=True and add the header manually. I believe we could just choose to require only the boolean and add the header transparently.

But it does appear to work if both gzip_enabled=True and headers={"Accept-Encoding":"gzip"} is provided to fido.fetch.

Reply:
Good call! I didn't want to make any assumptions about the request headers but I agree that developers would probably expect the accepted encoding to be implicitly updated